types/comtac/Cluey/lifecycle.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$id": "https://akenza.io/comtac/cluey/lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Lifecycle",
+  "type": "object",
+  "properties": {
+    "batteryLevel": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 100,
+      "title": "Battery charge",
+      "description": "The battery charge in percent.",
+      "unit": "%"
+    },
+    "batteryPowered": {
+      "title": "Battery powered",
+      "description": "Is the battery powered",
```

Comment:
Whether the battery is powered.

Reply:
I would rather formulate in passive voice than a question, but this is more stylistic